srcpkgs/doxygen/template

```diff
@@ -1,9 +1,10 @@
 # Template file for 'doxygen'
 pkgname=doxygen
 version=1.8.20
-revision=1
+revision=2
 build_style=cmake
-hostmakedepends="chroot-git perl python flex"
+configure_args="-DCMAKE_BUILD_TYPE=None"
```

Comment:
Perhaps it would be good to leave a comment for why this is needed?

Reply:
I think it's well-known that our cmake use `-DCMAKE_BUILD_TYPE=Release` which add `-O3` and some other `C{,XX}FLAGS`, I tried to change it once but some packages that try to be clever and do some weird things when `CMAKE_BUILD_TYPE` is not `Release`. Hence, I added them manually to those packages known to not broken.

Regarding "breaking masterdir", with `chroot-git` in `hostmakedepends`, `xbps-src` will try to remove it up-on cleanup. And it's a dependency of `base-chroot` then it'll be broken.

Reply:
Ok, got it. Thanks!